<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>netcfgfx_network_scanner_UI</class>
 <widget class="QDialog" name="netcfgfx_network_scanner_UI">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>285</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Choose a wireless network</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/netcfgfx_main_icon.png</normaloff>:/netcfgfx_main_icon.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="animated">
        <bool>false</bool>
       </property>
       <property name="allColumnsShowFocus">
        <bool>false</bool>
       </property>
       <property name="columnCount">
        <number>5</number>
       </property>
       <column>
        <property name="text">
         <string>ESSID</string>
        </property>
        <property name="textAlignment">
         <set>AlignLeft|AlignVCenter</set>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Signal</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Channel</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Mode</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Protocol</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="errorLabel">
       <property name="font">
        <font>
         <pointsize>7</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="refreshPushButton">
         <property name="text">
          <string>Refresh</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/recur.png</normaloff>:/recur.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>148</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="choosePushButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Choose</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/ok.png</normaloff>:/ok.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelPushButton">
         <property name="text">
          <string>Cancel</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/cancel.png</normaloff>:/cancel.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>netcfgfx_network_scanner_UI</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
